app/master/subjob.py

```diff
@@ -103,29 +98,6 @@ def atomic_commands(self):
         job_command = self.job_config.command
         return ['{} {}'.format(atom.command_string, job_command) for atom in self._atoms]
 
-    def get_atom_artifact_name(self, atom_id):
```

Comment:
That's a good point.

On a related note, would you say that it's not worth creating the extra static helper methods to generate the timing file/console output file/exit code file? Because it wouldn't be used in the scenario that we keep the instance method.